Dockerfile

```diff
@@ -68,6 +68,7 @@ ENV PS1="$(whoami)@$(hostname):$(pwd)\\$ " \
 RUN \
   echo "**** install base packages ****" && \
   dnf -y --setopt=install_weak_deps=False --best install \
+    busybox \
```

Comment:
Is there a reason for busybox over cron? Have we checked there aren't any side effects from installing busybox that could impact other installed GNU tools?

Reply:
> Is there a reason for busybox over cron? Have we checked there aren't any side effects from installing busybox that could impact other installed GNU tools?

Yes. Busybox is added to the PATH in an order that allows any other tools like GNU ones to take priority if they are installed. This is confirmed.

The reason to prefer busybox cron over other options is that we can universally use the same cron everywhere (except ubuntu/debian because they build and package busybox without cron so we have to use their packaged cron)